event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepository.java

```diff
@@ -82,6 +87,13 @@ public EventJdbcRepository(final EventInsertionStrategy eventInsertionStrategy,
         this.logger = logger;
     }
 
+//    public EventJdbcRepository(final DataSource dataSource) throws DatasourceException{
+//        if (dataSource== null){
+//            throw new DatasourceException("Data source is null");
+//        }
+//        this.dataSource = dataSource;
+//    }
+
```

Comment:
Commented out code can be deleted

Reply:
Reverted to old code no changes are required in this class